src/components/layout/MobileNavigation.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div
+    class="mobile-navigation clearfix"
+    :class="[
+      (open ? 'open' : ''),
+      ($route.name !== 'home' ? 'sticky' : ''),
+    ]"
+    v-show="$route.name !== 'home' || !isMobileNavigationHidden"
+  >
+    <div
+      class="close-navigation"
+      v-if="open"
+      @click="openNavigation(false)"
+    >
+      &#x2715;
+    </div>
+    <navigation/>
+    <div class="mobile-actions">
+      <img
+        class="tip"
+        :src="emptyHeart"
+        @click="toggleTipModal(true)"
+        v-if="!open && $route.name === 'home'"
```

Comment:
you can move the v-if=!open part in the parent element, seems more logical to me

Reply:
The trigger-navigation button has a different condition. There are 2 buttons that disappear if   we are not on the home page. The parent would hide all and is used for styling.

Reply:
i mean only !open but leave the route === home per item where applies

Reply:
Yes, you are right